install-dev/upgrade/upgrade.php

```diff
@@ -77,16 +77,10 @@
 }
 
 require_once(dirname(__FILE__).'/../init.php');
-Migrate::migrateSettingsFile();
+Upgrade::migrateSettingsFile();
 require_once(_PS_CONFIG_DIR_.'bootstrap.php');
 
 
-// set logger
-require_once(_PS_INSTALL_PATH_.'upgrade/classes/AbstractLogger.php');
-eval('abstract class AbstractLogger extends AbstractLoggerCore{}');
-require_once(_PS_INSTALL_PATH_.'upgrade/classes/FileLogger.php');
-eval('class FileLogger extends FileLoggerCore{}');
-
 $cacheDir = _PS_ROOT_DIR_.'/'.(_PS_MODE_DEV_ ? 'dev' : 'prod').'/log/';
 @mkdir($cacheDir, 0777, true);
 $logger = new FileLogger();
```

Comment:
It was already loaded in classes/log/AbstractLogger.php, same for FileLoggerCore.php